### PR TITLE
Pyramid example

### PR DESCRIPTION
**Pull request by bhoomit**
Pyramid example with README.

### CODE REVIEW

File: docs/examples/pyramid/base.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    def post_param(self, name):
+        return self.params.get(name)
+
+
+class BaseAuthController(object):
+
+    def __init__(self, request, site_adapter):
+        self.request = Request(request)
+        self.site_adapter = site_adapter
+        self.token_generator=Uuid4()
```
Comment (wndhydrnt) [inline docs/examples/pyramid/base.py]: Missing spaces surrounding `=`.

File: docs/examples/pyramid/base.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
+__author__ = 'Bhoomit'
```
Comment (wndhydrnt) [inline docs/examples/pyramid/base.py]: Other files do not include the `__author__` flag. Please remove it.

File: docs/examples/pyramid/impl.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
+__author__ = 'Bhoomit'
```
Comment (wndhydrnt) [inline docs/examples/pyramid/impl.py]: Other files do not include the `__author__` flag. Please remove it.

**Comment by wndhydrnt** [comment]
Great work! Just found some nits, otherwise it LGTM.
I'm looking forward to a working example.

**Comment by bhoomit** [comment]
@wndhydrnt will add one soon.

**Comment by wndhydrnt** [comment]
:+1:

**Pull request merged**